app/models/popular_list_set.rb

```diff
@@ -30,6 +30,8 @@ def search_response
       fields: SearchApiFields::POPULAR_BROWSE_SEARCH_FIELDS,
     }
     Services.cached_search(params)
+  rescue GdsApi::HTTPServerError
+    {}
```

Comment:
I wonder, should the rescued error be sent to Sentry, so we know something is going on?